data/entities.yaml

```diff
@@ -1053,6 +1053,21 @@
       3) (\x. 2*x) (\y. 3*y) == 9`.
   properties: [pickable]
   capabilities: [arith, sum, prod]
+- name: hyperloop
+  display:
+    attr: device
+    char: '∞'
+  description:
+    - |
```

Comment:
I'm not sure if this is just on my machine but the rendering for this entity is off?
![CleanShot 2024-06-06 at 22 35 13@2x](https://github.com/swarm-game/swarm/assets/15181803/36e71926-8522-4164-9c23-7e0ad5141fc2)

Is this intentional?

Reply:
It's not intentional, and in fact I think it is related to #1794.  Somehow each of those code snippets `rec t. T(t)` and `t = T(t)` is being pretty-printed onto multiple lines.

Reply:
Quite possibly.

Reply:
I have been trying to understand where this happens in the code and I have not been able to figure it out.  But in any case the issue is independent of this PR.  I created #1910 to track it.